web-registry/src/pages/ChooseCreditClassPage/ChooseCreditClassPage.tsx

```diff
@@ -1,12 +1,102 @@
 import React from 'react';
```

Comment:
This file used to be `ChooseCreditClass`. Since we are deleting `ChooseCreditClassLegacy`, I renamed this to `ChooseCreditClassPage` because that was just a wrapper for this file.

Reply:
This may look as if `ChooseCreditClass` was deleted, but really it was just renamed.